### PR TITLE
feat(google-recaptcha): added google recaptcha field

### PR DESCRIPTION
**Pull request by EpicIvo**

### CODE REVIEW

File: src/Captcha/index.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Component } from 'react';
+import { observer } from 'mobx-react';
+
+/**
+ * This component is necessary for the recaptcha element to appear.
+ */
+@observer
```
Comment (bartlangelaan) [inline src/Captcha/index.jsx]: Why is it an observer? It doesn't seem like this component does anything with observable props.

```suggestion
```

File: src/Webform/ThankYouMessage/styled/message.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 export default styled.h2`
-  background-color: ${p => p.theme.primaryColor};
+  background-color: transparent;
```
Comment (bartlangelaan) [inline src/Webform/ThankYouMessage/styled/message.js]: Just to check.. Isn't this the default? If so, we should remove this line of re-setting it's default value.

```suggestion
```

Comment (ianwensink) [reply src/Webform/ThankYouMessage/styled/message.js]: Yeah, this can probably be removed!

File: src/Captcha/index.jsx (hunk 1 above)
Comment (ianwensink) [reply src/Captcha/index.jsx]: Agreed

File: src/Webform/index.jsx [hunk 3]
```diff
@@ -105,6 +108,8 @@ class Webform extends Component {
     this.onSubmit = this.onSubmit.bind(this);
     this.submit = this.submit.bind(this);
 
+    this.recaptchaRef = React.createRef();
```
[the remark this answers is not in the document]
Comment (ianwensink) [reply src/Webform/index.jsx]: Let's use the old way of refs for now

**Pull request merged**